x/ibctransferhooks/module.go

```diff
@@ -0,0 +1,146 @@
+package ibctransferhooks
+
+import (
+	"encoding/json"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	cdctypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	capabilitytypes "github.com/cosmos/cosmos-sdk/x/capability/types"
+	channeltypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
+	porttypes "github.com/cosmos/ibc-go/v3/modules/core/05-port/types"
+	"github.com/cosmos/ibc-go/v3/modules/core/exported"
+	"github.com/gorilla/mux"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/spf13/cobra"
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	"github.com/persistenceOne/persistence-sdk/x/ibctransferhooks/keeper"
+	"github.com/persistenceOne/persistence-sdk/x/ibctransferhooks/types"
+)
+
+var (
+	_ module.AppModule      = Wrapper{}
+	_ module.AppModuleBasic = AppModuleBasic{}
+	_ porttypes.IBCModule   = Wrapper{}
+)
+
+type AppModuleBasic struct{}
+
+func (a AppModuleBasic) Name() string { return types.ModuleName }
+
+func (a AppModuleBasic) RegisterLegacyAminoCodec(_ *codec.LegacyAmino) {}
+
+func (a AppModuleBasic) RegisterInterfaces(_ cdctypes.InterfaceRegistry) {}
+
+func (a AppModuleBasic) DefaultGenesis(_ codec.JSONCodec) json.RawMessage {
+	return nil
+}
+
+func (a AppModuleBasic) ValidateGenesis(_ codec.JSONCodec, _ client.TxEncodingConfig, _ json.RawMessage) error {
+	return nil
+}
+
+func (a AppModuleBasic) RegisterRESTRoutes(_ client.Context, _ *mux.Router) {}
+
+func (a AppModuleBasic) RegisterGRPCGatewayRoutes(_ client.Context, _ *runtime.ServeMux) {}
+
+func (a AppModuleBasic) GetTxCmd() *cobra.Command {
+	return nil
+}
+
+func (a AppModuleBasic) GetQueryCmd() *cobra.Command {
+	return nil
+}
+
+type Wrapper struct {
+	AppModuleBasic
+	k keeper.Keeper
```

Comment:
should we just call it `keeper`?

Reply:
it stutters with imports, we can rename the import and call it keeper

Reply:
P.S. i want to rename Wrapper struct as AppModule... 
because it is a AppModule

Reply:
taking this up